### DISCUSSION TITLE
Purpose of `--subuidname` in rootless mode

### DISCUSSION THREAD

**Discussion by kaivol** [containers/podman#21147] [Q&A] [answered]
When using the `--subuidname` option in rootless mode, "the user needs to have the right to use the [given] mapping" ([source](https://docs.podman.io/en/latest/markdown/podman-run.1.html#subuidname-name)).

I am not particularly familiar with linux subuid stuff, but I can't think of a scenario where a rootless user would have the right to use a subuid map other than his own. As I understand it, the subuid map's name is equivalent to the name of the **_one and only_** user who can use it. 

Am I missing something, or is the documentation here misleading?

**Comment by rhatdan** [comment, marked as answer]
I think the range of UIDs have to be contained within the range of UIDs available to the user.

I am guessing something like the following would work.

```
dwalsh:100000:2000000
dwalsh1:100000:1000000
dwalsh2:100001:1000000
```
